pallets/scheduler/src/lib.rs

```diff
@@ -46,6 +53,15 @@
 //! If a call is scheduled using proxy or whatever mecanism which adds filter,
 //! then those filter will not be used when dispatching the schedule call.
```

Comment:
This documentation is a part of parity scheduler pallet, I do not think it would be appropriate to rewrite it.
"origin" and "root" are basic concepts, I don't think it's possible to understand at least something in the code without knowing the basics.